src/checks.rs

```diff
@@ -1407,6 +1412,9 @@ impl CheckKind {
             CheckKind::DoNotAssertFalse => "Do not `assert False` (`python -O` removes these \
                                             calls), raise `AssertionError()`"
                 .to_string(),
+            CheckKind::JumpStatementInFinally => "`return/continue/break` inside finally blocks \
```

Comment:
Can we include which of these cases was triggered directly in the rule? So that it's like `return inside finally blocks...` or as relevant?

Reply:
Got it, will fix